analytics/README.md

```diff
@@ -50,9 +76,42 @@ If you need to be added to the slack workspace or to the list of collaborators f
 3. From the side menu, select `OAuth & Permissions` and scroll down to the "OAuth tokens for your workspace" section
 4. Copy the "Bot user OAuth token" which should start with `xoxb` and paste it into your `.secrets.toml` file under the `slack_bot_token` variable.
 
-<img alt="Screenshot of slack app settings page with bot user OAuth token" src="./static/screenshot-slackbot-token.png" width=900>
+<img alt="Screenshot of slack app settings page with bot user OAuth token" src="./static/screenshot-slackbot-token.png" width=750>
 
-## Getting started
+## Using the make commands
+
+In most cases, the reports you'd like to run are already available as `make` commands, specified in our [`Makefile`](./Makefile)
+
+### Export data and run reports
+
+If want to run reports with the most recent data from GitHub, the easiest way to do it with the `make sprint-reports-with-latest-data`
```

Comment:
You probably mean something like `the easiest way to do it *is* with the`. Also period at end of sentence.

Reply:
Thanks for catching this! Updated it in this commit: [fix: typo in analytics/README](https://github.com/HHS/simpler-grants-gov/pull/835/commits/71914b004683b75e44d7c3c98472262ab48d0c27)